packages/mui-lab/src/internal/pickers/PickersPopper.tsx

```diff
@@ -52,6 +65,19 @@ const PickersPopperPaper = styled(Paper, { skipSx: true })<{
   }),
 }));
 
+const PickersPopperAction = styled(DialogActions, { skipSx: true })<{
```

Comment:
```suggestion
const PickersPopperAction = styled(DialogActions)<{
```

We decided to remove the `skipSx: true` with https://github.com/mui-org/material-ui/pull/30749